### ISSUE TITLE
ADRIFT 4 enhancement: Start game with initial command or disable GLK abbreviations initially?

### ISSUE THREAD

**Issue by kpeamon** [curiousdannii/emglken#30]
The SCARE interpreter has a few limitations. One of them is that it comes with some "smart" abbreviations, such as C for CLOSE, which is unfortunate in a game where you are given lettered options. Then the player can't choose option C by typing C as it is interpreted as CLOSE. This can easily be fixed by typing GLK ABBREVIATIONS OFF but the player is not supposed to know that, so it would be good if those could be off from the start.
 This specific problem is the case with the ADRIFT 4 game "The PK Girl".

**Comment by curiousdannii** [comment]
Does that setting also affect direction shortcuts or x for examine?

**Comment by kpeamon** [comment]
No, normal abbreviations works - it was extra unnecessary abbreviations which causes problems in a few games. can be turned back on with GLK ABBREVIATIONS ON

**Comment by curiousdannii** [comment]
So this is complicated by the fact that Scare actually has two abbreviation expanding functions. One of them is not specific to Glk and can be disabled by a compile time option added in https://github.com/garglk/garglk/pull/552

The others are just in Glk, defined here: https://github.com/garglk/garglk/blob/master/terps/scare/os_glk.c#L2573
They can be disabled by a runtime CLI argument (and then reenabled with "glk abbreviations on".)

A third option would be to add another compile time option that removes the weird ones like c/close, k/attack, p/open, r/drop.

I'll have to test it more myself to see which I think would be best.

**Comment by kpeamon** [comment]
So far I am only aware of one game where it is a problem so I was hoping for something that could be added to the URL only when nedded.
